.kokoro/continuous/common.cfg

```diff
@@ -20,6 +20,16 @@ env_vars: {
 }
 
 env_vars: {
-  key: "JOB_TYPE"
-  value: "test"
+  key: "PROJECT_NUMBER"
+  value: "945579214386"
+}
+
+env_vars: {
+  key: "BUCKET_NAME"
+  value: "crs-interactive-tutorials"
+}
+
+env_vars: {
+  key: "EVENTS_BUCKET_NAME"
+  value: "crs-interactive-tutorials"
```

Comment:
Same with this file (see https://github.com/googleapis/java-retail/blob/main/.kokoro/populate-secrets.sh)

Reply:
Hi, Kurtis, @kurtisvg, kould you please help me to understand, if I need to populate the secrets from the secrets manager  from service account: kokoro-trampoline@cloud-devrel-kokoro-resources.iam.gserviceaccount.com, how can I add our secrets there? Should I request assistance from some team?

Reply:
@chingor13 knows more about how to add secrets, although the correct process might just be to ask someone with permissions.

@lesv FYI

Reply:
done